submittedResponses.tf

```diff
@@ -0,0 +1,16 @@
+resource "aws_dynamodb_table" "submitted-responses-table" {
+  name           = "${var.env}-SubmittedResponses"
```

Comment:
i think to make this name look more consistent with the env prefix it should be `${var.env}-submitted-responses`